src/components/content-tab-assistant.tsx

```diff
@@ -228,11 +228,20 @@ const AuthenticatedView = memo(
 		isAssistantThinking: boolean;
 	} ) => (
 		<>
-			{ messages.map( ( message, index ) => (
-				<Message key={ index } id={ `message-${ index }` } isUser={ message.role === 'user' }>
-					{ message.content }
-				</Message>
-			) ) }
+			{ messages.map( ( message, index ) => {
+				const isLastAssistantMessage = index === messages.length - 1 && message.role !== 'user';
+				return (
+					<div
+						className={
+							isLastAssistantMessage ? 'animate-in slide-in-from-left zoom-in duration-300' : ''
+						}
+					>
```

Comment:
I noticed that React is warning with the following message:
```
Warning: Each child in a list should have a unique "key" prop.
```

This could be solved by moving the `key` prop from the `Message` component to the wrapper `div`.

Reply:
Thaks. I missed the warning :smile: